### ISSUE TITLE
Code Block Python Markdown triple quotes broken

### ISSUE THREAD

**Issue by nylocx** [mattermost/mattermost-server#16356]
#### Summary
If you add multi lines string to the python code block markdown only the first line is formatted as a string.

#### Steps to reproduce
Add a Python code block like this.
````
```python
multi_line_str="""This is
a multi
line
string"""
```
````

#### Expected behavior
It should look somewhat like this
```python
multi_line_str="""This is
a multi
line
string"""
```

#### Observed behavior (that appears unintentional) 
it actually looks like this:
![image](https://user-images.githubusercontent.com/3329124/99945725-6f7fc480-2d75-11eb-9f9d-3717a6d53807.png)

#### Possible fixes
Fix the code block parser for this syntax construct.

**Comment by amyblais** [comment]
Opened a ticket https://mattermost.atlassian.net/browse/MM-30879

**Comment by amyblais** [comment]
This was actually fixed in v5.28.
